### DISCUSSION TITLE
Saleor vs Shopfiy Comparison

### DISCUSSION THREAD

**Discussion by bytearchive** [saleor/saleor#6582] [Q&A]
### What I'm trying to achieve

I am deciding if it is better to setup Saleor + stripe + shipping etc on my own or just use Shopify basic plan.

It would seem that this question would be interesting to anyone starting an online shop.

Has anyone done this comparison? is there a page in documentation that shows this comparison.

**Comment by karolkielecki** [comment]
@bytearchive Hi there!

That’s a good question, thanks for asking. Unfortunately, I don’t think there is such a document comparing both platforms. 

Saleor provides a modern development stack and comes with a SasS offer. It’s the fastest-growing **open-source** platform, maintained by the business and supported by the community. It’s powered by GraphQL, Python, React and PWA. We believe that innovation comes through integrations, so freedom of development is an essential part of our platform. 

It depends on what you are looking for, so here are a few advantages of using Saleor:
- Headless. Apart from performance, the API-first approach is the most flexible way of building complex web solutions, including e-commerce. Exposing and integrating APIs leaves the doors open for future integrations. No more resorting to clunky monolithic platforms.
- It’s GraphQL-first. It’s the most efficient and modern way of writing web APIs.
- Building projects around APIs makes it not only efficient with GraphQL but also improves its stability through the isolation of features. It eliminates the possibility that issues in one feature will affect the functionality of another.
- Front-end developers don’t need to wait for fully finished code in the back-end.

It’s worth mentioning that Saleor 3.0 will soon be released which will introduce a multi-channel/multi-currency feature. You can find our roadmap here: https://github.com/mirumee/saleor/projects/12

I recommend you try the saleor-platform repo: https://github.com/mirumee/saleor-platform.  It’s been designed only for local development, but it’s a good starting point with Saleor. 

And if you have more questions, feel free to reach us on Spectrum: https://spectrum.chat/saleor

Best,
✌️

**Comment by karolkielecki** [comment]
You can also check the [new stores powered by Saleor here](https://spectrum.chat/saleor/general/our-new-saleor-stores-are-online~3db5acfb-4257-4a67-a649-cf5967ee7f99) :)
